src/utils/formatMeasurement.ts

```diff
@@ -0,0 +1,7 @@
+export const formatMeasurement = (value: number, measureUnit: string) => {
+  const baseFormatted = `${value} ${measureUnit}`;
+
+  if (value === 0) return "";
+  if (value === 1) return baseFormatted;
+  if (value > 1) return `${baseFormatted}s`;
+};
```

Comment:
The logic here doesn't seem correct? I believe usually both the `0` and `> 1` cases should be plural.

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1302/commits/61c8d31c7eecfeae9b1debbdc10b22f31a630afe